docs/core-concepts/run-udfs/run.mdx

```diff
@@ -251,6 +251,19 @@ A [`fused.run()`](/python-sdk/top-level-functions/#run) call will require the fo
 3. [Optional] Any protected arguments as seen in the dedicated [API docs page](/python-sdk/top-level-functions/#run) (if applicable). These include for example:
    - `bounds` -> A geographical bounding box (as a list of 4 point: `[min_x, min_y, max_x, max_y]`) defining the area of interest.
    - `cache_max_age` -> The maximum [age of the UDF's cache](/core-concepts/cache/#defining-your-cache-lifetime-cache_max_age).
+   - `dtype_out_raster` -> The output raster format. Defaults to "tif". Change this to "png" for a simple preview of your image.
+
+#### Examples of using parameters
+
+Changing the default cache max age:
+```python showLineNumbers
+fused.run("UDF_CDLs_Tile_Example", cache_max_age='1d')
+```
+
+Calling a public UDF that returns a raster, we can change the output format to png for a simple preview:
+```python showLineNumbers
+fused.run("UDF_CDLs_Tile_Example", dtype_out_raster='png')
+```
```

Comment:
this doesn't give a preview image though, it's if you don't have the requirements